python/GangaAtlas/Lib/Athena/Athena.py

```diff
@@ -1004,6 +1004,13 @@ def prepare(self, force=False, **options):
 
         logger.info('Found Working Directory %s',self.userarea)
         logger.info('Found ATLAS Release %s',self.atlas_release)
+
+        # check for nightlies
+        if "AtlasBuildStamp" in os.environ:
+            # change production as the release found above is one ahead of the actual
+            logger.info('Found ATLAS Nightly release. Setting atlas_production appropriately')
```

Comment:
I prefer to be a bit verbose with the config extraction just to aid debugging in the future.